kiliautoml/models/_pytorchvision_image_classification.py

```diff
@@ -104,6 +104,13 @@ def train(
         splits["train"], splits["val"] = train_test_split(
             range(len(labels)), test_size=0.2, random_state=42
         )
+        label_train = [labels[i] for i in splits["train"]]
```

Comment:
note for later: this should be generalized to all the tasks, with a common method (for splitting and checking the label repartition)

Reply:
yes. I think you are right. Next week's workshop with Raphaël will help us decide what to choose.